src/components/AccountForms/RegistrationForm/RegistrationForm.styles.js

```diff
@@ -0,0 +1,152 @@
+import { makeStyles } from '@material-ui/core/styles';
+import '@fontsource/roboto';
```

Comment:
Is this dependency necessary?

Reply:
Yes the fonts in the form and the chip didn't match before

Reply:
👍